spec/decorators/offence_decorator_spec.rb

```diff
@@ -108,4 +109,48 @@
       it { is_expected.to eql('plea is a string') }
     end
   end
+
+  describe '#mode_of_trial_reason_list' do
+    subject(:call) { decorator.mode_of_trial_reason_list }
+
+    context 'when 1 or more mode of trial reasons exist' do
+      let(:mode_of_trial_reason_array) do
+        [{ code: '4',
+           description: 'Defendant elects trial by jury' },
+         { code: '2',
+           description: 'Indictable-only offence' }]
+      end
+
+      before do
+        allow(offence).to receive(:mode_of_trial_reasons).and_return(mot_reason_ostruct_collection)
+      end
+
+      it { is_expected.to eql('Defendant elects trial by jury<br>Indictable-only offence') }
+    end
```

Comment:
The most common response we are too expect is one reason so it might be good to cover that as a separate specific test first.

i.e.
```
context 'when exactly 1 mode of trial reason exists' do

```
you could nest the multiple and singluar contexts within 'when reasons exists' to reduce repetition - hppy to pair on it

Reply:
👍 have added 'when exactly 1 mode of trial reason exists' and 'when more than 1 mode of trial reason exists' contexts.